### DISCUSSION TITLE
Set border to 0px/transparent on AreaClosed Chart

### DISCUSSION THREAD

**Discussion by LukasGirsch** [airbnb/visx#1399] [Q&A] [answered]
Hi there,
is there a way to set the borders to 0px or a different color other than the stroke color?

I would like to add a pattern under the line without masking the Chart itself to hide the border.

![image](https://user-images.githubusercontent.com/6472013/147272213-a06ac999-a29d-4167-a44e-16f10e773e45.png)

**Comment by hshoff** [comment, marked as answer]
Hi @LuKaa,

You can use the standard svg attributes to modify the stroke.

can set the [stroke width](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-width) to `0`:
```jsx
<AreaClosed
  // ...
  strokeWidth={0}
/>
```
or set the [stroke](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke) color to transparent:
```jsx
<AreaClosed
  // ...
  stroke="transparent"
/>
```

Then add a `<LinePath />` after the `<AreaClosed />` to render the line on top of the area.

```jsx
<AreaClosed
  // ...
  strokeWidth={0}
/>
<LinePath
  // ...
  strokeWidth={3}
  stroke="blue"
/>
```

**Reply by LukasGirsch** [reply]
Thank you @hshoff ,
I don't know why I did not think about this but it does totally solve my issue.

Kind regards and Merry Christmas.
